styling.css

```diff
@@ -0,0 +1,59 @@
+thead, tfoot {
+    font-weight: bold;
+    border: solid black 1px;
+    position: relative;
+}
+
+thead::before, tfoot::before {
```

Comment:
Docela kreativni reseni uprimne :) Jestli to slouzi k dosazeni ramecku u tbody/thead, pak bych to udelal napr.:
```css
table { border-collapse: collapse; }
thead tr , tfoot tr {
    font-weight: bold;
    border: solid black 1px;
}
```

tyhle `::before` (radek 7 az 16 ) smazat, a podobne `position:relative` na radku 4 taky smazat 

U tabulky, vychozi `border-collapse` je `seperate` a to ma vliv na ramecky a zarovnani. 

Tady kdyz si zacinas hrat s `position`, tak se to muze casem komplikovat :) Rozhodne tohle neberu to jako chybu, jen mozna moc slozite reseni